src/test/librbd/journal/test_mock_Replay.cc

```diff
@@ -433,6 +450,47 @@ TEST_F(TestMockJournalReplay, AioWriteSame) {
   ASSERT_EQ(0, on_safe.wait());
 }
 
+TEST_F(TestMockJournalReplay, AioCompareAndWrite) {
```

Comment:
Please add a test for -EILSEQ since that is an error we should expect to see a lot on replay.